{"name": "test_101_110[全部输入，输入账号不正确]", "status": "passed", "parameters": [{"name": "username", "value": "'17854225746'"}, {"name": "password", "value": "'123456'"}, {"name": "verify_code", "value": "'8888'"}, {"name": "expectedtext", "value": "'账号不存在!'"}], "start": 1596270186001, "stop": 1596270207939, "uuid": "d52ec43f-cd3c-4592-b772-facf2facad05", "historyId": "201406f958edc89ea2d16b0e2edc26e8", "testCaseId": "674c8a70a912fd8799a90691a3422221", "fullName": "cases.登录.test_login.Test_login#test_101_110", "labels": [{"name": "story", "value": "登录失败测试"}, {"name": "feature", "value": "登录功能"}, {"name": "parentSuite", "value": "cases.登录"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test_login"}, {"name": "host", "value": "PC-20180809BGFX"}, {"name": "thread", "value": "17008-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "cases.登录.test_login"}]}